### ISSUE TITLE
Tuple IDs not working on NetworkX

### ISSUE THREAD

**Issue by marimeireles** [QuantStack/ipycytoscape#86]
Thanks to some comments on #70 I tried to add some tuples as IDs to nodes and tried to connect them to create a graph.

It's not working.

Here is some reproducible code to run in a notebook:

```
import networkx
import matplotlib
import ipycytoscape

undirected = ipycytoscape.CytoscapeWidget()
undirected.graph.add_graph_from_networkx(G)
undirected
G2 = nx.Graph()
G2.add_node((0, 1))
G2.add_node((0,2))
G2.add_edge((0,1), (0,2))
nx.draw(G2)
undirected.graph.add_graph_from_networkx(G2)
```

This code will produce the following result:

![Screenshot_2020-06-16_21-01-47](https://user-images.githubusercontent.com/17600982/84817484-8f5fcf00-b004-11ea-9084-2d08f12ad611.png)

Meaning that this is a valid output for NetworkX but ipycytoscape is not able to render it. Doesn't throw errors
I tried to convert the node to a string:

[link to node]

But didn't work, even though node is successfully converted to string, I'm getting this js error.
```
Error setting state: Can not create edge `dce70ec3-36b9-4bbc-a2d3-6b4a0365b5cf` with nonexistant source `1,2`
```

**Comment by ianhi** [comment]
Confusingly I get a different javascript error
![image](https://user-images.githubusercontent.com/10111092/84838162-ace26800-b007-11ea-85d7-3d05b85666cc.png)
I did however find that this code worked:
```python
import networkx as nx
import ipycytoscape

undirected = ipycytoscape.CytoscapeWidget()
G2 = nx.Graph()
G2.add_node(str((0, 1)))
G2.add_node(str((0,2)))
G2.add_edge(str((0,1)), str((0,2)))
nx.draw(G2)
undirected.graph.add_graph_from_networkx(G2)
undirected
```
giving:
![image](https://user-images.githubusercontent.com/10111092/84838276-fa5ed500-b007-11ea-9faf-88ed6b72c25e.png)

**Comment by marimeireles** [comment]
Okay, will give another try in a different env. it seems like I broke mine yesterday night somehow.

**Comment by ianhi** [comment]
fixed by #118
